Modules/MessageUtils/PaginatedEmbed.js

```diff
@@ -1,75 +1,76 @@
 const { Colors, PageEmojis } = require("../../Internals/Constants");
 
 class PaginatedEmbed {
-	constructor (msg, descriptions = [], embed = null, images = [], fields = [], thumbnails = []) {
-		this.client = msg.client;
-		this.originalMsg = msg;
-
+	/**
+	 * After creating a PaginatedEmbed call `#init()` to set it up and start listenig for reactions.
+	 *
+	 * @param originalMsg 	The original message that created this paginated embed.
```

Comment:
None of these jsdocs have typedefs (`@param {Message} originalMsg`)

Reply:
What should I use for the second argument? Create a type? This isn't really meant to be a perfect documentation, this is just a little help for us. Other people won't be touching this anyways.